autoload/go/cmd.vim

```diff
@@ -83,12 +83,28 @@ function! go#cmd#Run(bang, ...)
         exe 'make!'
     endif
 
-    cwindow
-    let errors = getqflist()
+    let qflist = getqflist()
+    let errors = []
+    let is_readable = {}
+    for item in qflist
+        let filename = bufname(item.bufnr)
+        if !has_key(is_readable, filename)
+            let is_readable[filename] = filereadable(filename)
+        endif
+        if is_readable[filename]:
+            call add(errors, item)
+        endif
+    endfor
+    for k in keys(filter(is_readable, '!v:val'))
+        echo "vim-go: " | echohl Identifier | echon "[run] Dropped file " | echohl Constant | echon  '"' . k . '"'
+        echohl Identifier | echon " from QuickFix list (not readable in working directory)" | echohl None
```

Comment:
Remove the word `file`. We are actually dropping a text here which is not a file. Also the sentence `not readable in working directory` could be better when we rename it to: `(nonvalid file)` (or a better wording?). So in the end it would something like:

```
[run] Dropped "read 0 byte, err read tcp 127.0.0.1:8800: i/o timeout: []" from QuickFix list (nonvalid file)
```